src/pages/index.tsx

```diff
@@ -27,37 +30,105 @@ const IndexPage: React.FC<PageProps> = ({ location }) => {
+        imageAlt="TestAlt"
+        imageCredits=""
+        title="RSV-Baukasten"
+        caption="Projektansatz"
+      >
+        <p>
+          Mit dem RSV-Baukasten entsteht ein modulares System zur Durchführung
+          von Online-Beteiligungsformaten und Bürgerinformation zu
+          Radschnellverbindungen, das Bedarfe und Wissen konstruktiv in
+          Planungsprozesse einfließen lässt. Grundlage ist ein dynamisches
+          Online-Kartenformat das Informationen zu den geplanten
+          Radschnellverbindungen übersichtlich aufbereitet und
+          Beteiligungsinstrumente daran anbindet.
+        </p>
+        <b>Zentrale Ziele des geplanten RSV-Baukastens:</b>
+        <ul className="list-inside list-disc">
+          <li>
+            Passgenaue Online-Beteiligungsformate zur effektiven Einbindung der
+            Bürger:innen bei RSV-Planungen.
+          </li>
+          <li>
+            Verbesserte Kommunikation von RSV-Planungen in der Öffentlichkeit
+            und Steigerung der Akzeptanz von Planungsprojekten.
+          </li>
+          <li>
+            Beschleunigung der Errichtung von Radschnellverbindungen durch
+            zentralen Informationsaustausch und Kommunikationsschnittstellen auf
+            Ebene der Planenden.
+          </li>
+        </ul>
+      </TextWithImage>
+      <TextWithImage
+        imageUrl="https://images.unsplash.com/photo-1520923642038-b4259acecbd7?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=500&q=80"
+        imageAlt=""
+        imageCredits=""
+        title="Kontakt"
+        caption="Hintergrund"
+      >
+        <p>
+          Wenn Sie Interesse an einer Zusammenarbeit im Rahmen des Projektes
+          oder sich weitere Hintergründe zum Projekt wünschen, schreiben Sie mir
+          an{' '}
+          <ExternalLink href="mailto:hekele@radschnellverbindungen.info">
+            <b>hekele@radschnellverbindungen.info</b>
+          </ExternalLink>
+        </p>
 
-      2017 beschloss das baden-württembergische Verkehrsministerium, drei Radschnellverbindungen zu realisieren, bei denen das Land Bauherr sein und die Baulast übernehmen soll: Heidelberg-Mannheim, Heilbronn-Neckarsulm-Bad Wimpfen und Stuttgart-Esslingen-Plochingen. Für diese drei „Pilotstrecken“ wurde prioritär mit der Planung und dem Bau begonnen.[2][3] Laut einer Pressemitteilung des Ministeriums für Verkehr vom September 2018 wurden in einer landesweiten Potentialanalyse „32 Radschnellwege mit einer Gesamtlänge von 500 Kilometer im vordringlichen Bedarf identifiziert. Weitere 20 Strecken könnten sich durch lokalspezifische Faktoren als würdig für eine Radschnellverbindung erweisen.“ Insgesamt sollen bis 2025 zunächst zehn Radschnellwege in Baden-Württemberg realisiert werden.[4]"
-          title="Hintergrund"
-        />
-      </div>
+        <p className="mt-5 text-black">
+          <b>Boris Hekele, Projektleiter</b>
```

Comment:
```suggestion
          <strong>Boris Hekele, Projektleiter</strong>
```

Die `<b>` tun mir sematisch etwas weh :-). Oben bei der Email ist es noch okisch weil reines styling. Aber hier kann Semantik ran. Bessere SEO und bessere Barrierefreiheit.